tests/rule-engine/fc-config.scm

```diff
@@ -1,10 +1,21 @@
 ;
 ; Configuration file for the example crisp rule base system (used by
 ; fc.scm)
-;
-; Before running any inference you must load that file
 
-; Load the rules
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Load required modules and utils ;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(use-modules (opencog))
+(use-modules (opencog rule-engine))
+
+(load-from-path "utilities.scm")
+(load-from-path "av-tv.scm")
```

Comment:
Oh, I've just tried, the problem is that it's only true if the modules are installed on the system. So these are still useful if one wants to run these unit tests without installing opencog.